File: src/python/Player.py
class Player:
    def __init__(self, player_type):
        self.player_type = player_type

    def move(self):
        if self.player_type == "Warrior":
            return "Warrior moves forward"
        elif self.player_type == "Mage":
            return "Mage teleports"
        elif self.player_type == "Rogue":
            return "Rogue sneaks"
        else:
            raise ValueError("Invalid player type")

File: src/main/java/com/scrumcenter/csd/refactoring/polymorphism/Player.java
package com.scrumcenter.csd.refactoring.polymorphism;

public class Player {
    String playerType;

    public Player(String playerType) {
        this.playerType = playerType;
    }

    public String move() {
        return switch (playerType) {
            case "Warrior" -> "Warrior moves forward";
            case "Mage" -> "Mage teleports";
            case "Rogue" -> "Rogue freaks";
            default -> throw new IllegalArgumentException("Invalid player type");
        };
    }
}